source/Stargate-Examples-Tests/SouthAmericanCurrenciesRESTfulControllerTest.class.st

```diff
@@ -19,6 +19,30 @@ SouthAmericanCurrenciesRESTfulControllerTest >> setUpResourceController [
 	resourceController := SouthAmericanCurrenciesRESTfulController new
 ]
 
+{ #category : #tests }
+SouthAmericanCurrenciesRESTfulControllerTest >> testCantGetArgentinePesoAskingForUnsupportedLanguage [
+
+	| request |
+
+	request := self
+		requestToGETResourceIdentifiedBy: 'ARS'
+		accepting: resourceController currencyVersion1dot0dot0MediaType.
+	request setAcceptLanguage: 'de'.
+
+	self
+		should: [ resourceController currencyBasedOn: request within: self newHttpRequestContext ]
+		raise: HTTPNotAcceptable
+		withExceptionDo: [ :error | 
+			self
+				assert: error messageText
+					equals: 'Cannot generate a response entity with acceptable characteristics.';
+				assert: error code equals: 406;
+				assert: error allowedLanguageTags size equals: 2;
```

Comment:
Couldn't we change size and each content for an assertion for collection contents?

Reply:
I've tried it and is less legible